src/cript/api/api.py

```diff
@@ -463,16 +470,13 @@ def save(self, project: Project) -> None:
         None
             Just sends a `POST` or `Patch` request to the API
         """
-        # TODO work on this later to allow for PATCH as well
-        response = requests.post(url=f"{self._host}/{project.node_type.lower()}", headers=self._http_headers, data=project.json)
+        response: Dict = requests.post(url=f"{self._host}/{project.node_type.lower()}", headers=self._http_headers, data=project.json).json()
 
-        response = response.json()
-
-        # if htt response is not 200 then show the API error to the user
+        # if http response is not 200 then show the API error to the user
         if response["code"] != 200:
             raise CRIPTAPISaveError(api_host_domain=self._host, http_code=response["code"], api_response=response["error"])
```

Comment:
I think the way I originally did this works, but a better/cleaner way for the future might be to use the requests function to do this

```python
    response.raise_for_status()  # Raises an HTTPError if the status code is not 200
````

but not fully sure about it, it needs more research and testing to be sure it works correctly and the error it gives are good

Reply:
yeah, something for later to consider.